examples/multisig-example/src/main/kotlin/network/finschia/sdk/example/CreateValidator.kt

```diff
@@ -0,0 +1,266 @@
+/**
+ * Copyright 2023 Finschia Foundation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package network.finschia.sdk.example
+
+import network.finschia.sdk.legacymultisig.*
+import com.google.protobuf.ByteString
+import com.google.protobuf.kotlin.toByteString
+import cosmos.base.v1beta1.CoinOuterClass
+import cosmos.staking.v1beta1.Tx
+import io.grpc.ManagedChannelBuilder
+import kotlinx.serialization.json.*
+import network.finschia.sdk.account.HDWallet
+import network.finschia.sdk.account.KeyWallet
+import network.finschia.sdk.base.protoDecimalToJson
+import org.bouncycastle.jcajce.provider.digest.SHA256
+import java.util.*
+
+class MultiSigMsgCreateValidator {
+    companion object {
+        fun createMsgCreateValidator(
+            deliAddr: String,
+            opAddr: String,
+            amount: CoinOuterClass.Coin,
+            pubKey: String,
+        ): Tx.MsgCreateValidator {
+            val pubKeyBytes = Base64.getDecoder().decode(pubKey).toByteString()
+            val pk = com.google.protobuf.any {
+                typeUrl = "/cosmos.crypto.ed25519.PubKey"
+                value = pubKeyBytes.toPubKeyProtoValue()
+            }
+
+            val msgCreateValidator = cosmos.staking.v1beta1.msgCreateValidator {
+                description = cosmos.staking.v1beta1.description {
+                    moniker = "sim"
+                }
+                commission = cosmos.staking.v1beta1.commissionRates {
+                    rate = "100000000000000000"             // 0.1 * 10^18
+                    maxRate = "200000000000000000"          // 0.2 * 10^18
+                    maxChangeRate = "1"                     // 0.000000000000000001 * 10^18
+                }
+                minSelfDelegation = "1"
+                delegatorAddress = deliAddr
+                validatorAddress = opAddr
+                pubkey = pk
+                value = amount
+            }
+            return msgCreateValidator
+        }
+
+        private fun ByteString.toPubKeyProtoValue(): ByteString {
```

Comment:
Can't we use finschia-proto rather than creating the values by yourself?
In Cosmjs pubkey [test code](https://github.com/cosmos/cosmjs/blob/main/packages/proto-signing/src/pubkey.spec.ts#L13), this creates proto value for test. but in real code it uses [cosmjs-types code](https://github.com/cosmos/cosmjs/blob/main/packages/proto-signing/src/pubkey.ts#L40) which is same library with finschia-proto.

Reply:
I can't find `encode`, `decode` functions in finschia-proto kotlin part though.

Reply:
I wanted to use proto-type to make value but I wasn't able to find something like [cosmjs-types code](https://github.com/cosmos/cosmjs/blob/main/packages/proto-signing/src/pubkey.ts#L40).